components/airtable-demo.js

```diff
@@ -0,0 +1,14 @@
+import Airtable from 'airtable'
+
+var base = new Airtable({ apiKey: "keyQ54SEzABIZEm9F" }).base(
+  "appv6M4U4ckXcHzhU"
```

Comment:
Don't export API keys in the source code. Use environment variables.

Reply:
Will see to it online, will ask you if you if reqd.